### DISCUSSION TITLE
Welcome to matrix-commander Discussions!

### DISCUSSION THREAD

**Discussion by 8go** [8go/matrix-commander#69] [Announcements]
## 👋 Welcome!
  Github Discussions is a place for you to connect with other members of the `matrix-commander` community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Please be nice and friendly.

  To get started, if you feel like it, comment below with an introduction of yourself and tell us about 
  what you do with `matrix-commander` and about your vision to grow this community. 

  Welcome :clap: Bienvenue :clap: Ciao :clap: Hallo :clap: 
  Happy to see you here :+1:

**Comment by xurizaemon** [comment]
👋🏼 hi there, I'm chiefly interested in using Matrix Commander as a tool to surface the outcome of automated actions (particularly Gitlab CI) to a workplace Matrix environment. Really grateful for your work on Matrix Commander and the way you ask questions and learn in public. Thanks @8go!

**Reply by 8go** [reply]
Thanks @xurizaemon  for your kind words. Using `matrix-commander` in the context of GitLab CI seems to be a good use-case.

For those interested in the Gitlab CI use-case, please have a closer look at Issue #81 . An useful example is given there.
